go-controller/pkg/ovn/policy_test.go

```diff
@@ -107,24 +108,32 @@ func (n kNetworkPolicy) addLocalPodCmds(fexec *ovntest.FakeExec, networkPolicy *
 	n.addDefaultDenyPGCmds(fexec, networkPolicy)
 }
 
-func (n kNetworkPolicy) addNamespaceSelectorCmds(fexec *ovntest.FakeExec, networkPolicy *knet.NetworkPolicy, namespace string) {
+func (n kNetworkPolicy) addNamespaceSelectorCmds(fexec *ovntest.FakeExec, networkPolicy *knet.NetworkPolicy, namespace string, logSeverity ...string) {
 	as := []string{}
 	if namespace != "" {
 		as = append(as, namespace)
 	}
 
+	logSev := "info"
+	isLogEnabled := "false"
+	if len(logSeverity) > 0 {
+		logSev = logSeverity[0]
+		isLogEnabled = "true"
+	}
+
+	portGroup := hashedPortGroup(fmt.Sprintf("%s_%s", networkPolicy.Namespace, networkPolicy.Name))
 	for i := range networkPolicy.Spec.Ingress {
 		ingressAsMatch := asMatch(append(as, getAddressSetName(networkPolicy.Namespace, networkPolicy.Name, knet.PolicyTypeIngress, i)))
 		fexec.AddFakeCmdsNoOutputNoError([]string{
 			fmt.Sprintf("ovn-nbctl --timeout=15 --data=bare --no-heading --columns=_uuid find ACL external-ids:l4Match=\"None\" external-ids:ipblock_cidr=false external-ids:namespace=%s external-ids:policy=%s external-ids:Ingress_num=%v external-ids:policy_type=Ingress", networkPolicy.Namespace, networkPolicy.Name, i),
-			"ovn-nbctl --timeout=15 --id=@acl create acl priority=" + types.DefaultAllowPriority + " direction=" + types.DirectionToLPort + " match=\"ip4.src == {" + ingressAsMatch + "} && outport == @a14195333570786048679\" action=allow-related log=false severity=info meter=acl-logging name=" + networkPolicy.Namespace + "_" + networkPolicy.Name + "_" + strconv.Itoa(i) + " external-ids:l4Match=\"None\" external-ids:ipblock_cidr=false external-ids:namespace=namespace1 external-ids:policy=networkpolicy1 external-ids:Ingress_num=0 external-ids:policy_type=Ingress -- add port_group " + fakePgUUID + " acls @acl",
+			"ovn-nbctl --timeout=15 --id=@acl create acl priority=" + types.DefaultAllowPriority + " direction=" + types.DirectionToLPort + " match=\"ip4.src == {" + ingressAsMatch + "} && outport == @" + portGroup + "\" action=allow-related log=" + isLogEnabled + " severity=" + logSev + " meter=acl-logging name=" + networkPolicy.Namespace + "_" + networkPolicy.Name + "_" + strconv.Itoa(i) + " external-ids:l4Match=\"None\" external-ids:ipblock_cidr=false external-ids:namespace=namespace1 external-ids:policy=" + networkPolicy.GetName() + fmt.Sprintf(" external-ids:Ingress_num=%d", i) + " external-ids:policy_type=Ingress -- add port_group " + fakePgUUID + " acls @acl",
```

Comment:
I've had to fix these since they were only working for hard-coded values (namespace / policy names), etc.

Reply:
I think It's fine to keep all the unit testing changes in a single commit, just be sure to add anything you want to clarify in the commit message